docs/tutorials/zq-primer.md

```diff
@@ -17,20 +17,20 @@ pulled from the GitHub API.
 
 If you want to follow along on the command line,
 just make sure the `zq` command [is installed](../../README.md#quick-start)
-as well as [jq](https://stedolan.github.io/jq/).
+as well as [`jq`](https://stedolan.github.io/jq/).
 
 ## But JSON
 
 While `zq` is based on a new type of [data model called Zed](../formats/zed.md),
-it just so happens to be a superset of JSON.
+Zed just so happens to be a superset of JSON.
```

Comment:
I know the back-to-back comma-separated "Zed" is may seem a little odd, but as originally written, I felt like the "it" could have been read as a reference to `zq` _or_ to Zed, hence better to be explicit since this is intro text for many readers.